docs/configuration/plugins/syslogng-outputs/loki.md

```diff
@@ -82,5 +82,3 @@ Default: -
 Template for customizing the log message format. 
 
 Default: -
-
```

Comment:
[nit]
Is this generated? If so, how about fixing at the source?

Reply:
accidental ...